processing/src/main/java/io/druid/query/filter/InDimFilter.java

```diff
@@ -38,9 +39,14 @@
 {
   private final List<String> values;
   private final String dimension;
+  private final ExtractionFn extractionFn;
 
   @JsonCreator
-  public InDimFilter(@JsonProperty("dimension") String dimension, @JsonProperty("values") List<String> values)
```

Comment:
sure, you can file an issue for anything you don't think is appropriate for this PR.